### PR TITLE
Implement download links in docs theme

### PR DESCRIPTION
**Pull request by Firehawke**
This adds a pop-up bar in the bottom left corner with links to the PDF and EPUB versions of the documentation.
Try to avoid clobbering the ``versions.html`` file on theme update.

### CODE REVIEW

File: docs/themes/sphinx_rtd_theme/theme.conf [hunk 1]
```diff
@@ -18,3 +18,4 @@ prev_next_buttons_location = bottom
 style_external_links = False
 style_nav_header_background =
 vcs_pageview_mode =
+mamedevorg = True
```
Comment (cuavas) [inline docs/themes/sphinx_rtd_theme/theme.conf:21]: This file is missing the final newline.

File: docs/themes/sphinx_rtd_theme/versions.html [hunk 2]
```diff
@@ -32,3 +32,18 @@
...
+  <div class="rst-versions" data-toggle="rst-versions" role="note" aria-label="{{ _('Downloads') }}">
+    <span class="rst-current-version" data-toggle="rst-current-version">
+      <span class="fa fa-book">Downloads</span>
+      <span class="fa fa-caret-down"></span>
+    </span>
+    <div class="rst-other-versions">
+      <dl>
+          <dd><a href="https://docs.mamedev.org/_files/MAME.pdf">PDF</a></dd>
+          <dd><a href="https://docs.mamedev.org/_files/MAME.epub">EPUB</a></dd>
+      </dl>
```
Comment (cuavas) [inline docs/themes/sphinx_rtd_theme/versions.html:46]: The `dd` elements are indented two levels instead of one.

**Comment by cuavas** [comment]
I do wonder if there’s a way to set the conf variable in the environment so it won’t be set on local builds.

File: docs/themes/sphinx_rtd_theme/theme.conf (hunk 1 above)
Comment (Firehawke) [reply docs/themes/sphinx_rtd_theme/theme.conf]: Noticed it right after the PR was made. I'll go ahead and do corrections next chance I get to sit down and work on it.

File: docs/themes/sphinx_rtd_theme/versions.html (hunk 2 above)
Comment (Firehawke) [reply docs/themes/sphinx_rtd_theme/versions.html]: Yeah, will fix as part of next PR.

**Comment by Firehawke** [comment]
> I do wonder if there’s a way to set the conf variable in the environment so it won’t be set on local builds.

That's a good question. It'll require a deeper dive into Sphinx than I've needed to do to date, so it'll come in a later PR if it's possible. As it is, this 'fix' was about the best I could come up with in a reasonable amount of time.

**Pull request merged**